app/src/main/java/com/kickstarter/ui/viewholders/NativeCheckoutRewardViewHolder.kt

```diff
@@ -169,12 +169,14 @@ class NativeCheckoutRewardViewHolder(private val view: View, val delegate: Deleg
                 .compose(observeForUI())
                 .subscribe { this.view.reward_check.setBackgroundResource(it) }
 
-        view.reward_pledge_button.setOnClickListener {
+        this.view.reward_pledge_button.setOnClickListener {
```

Comment:
Cool! TIL in Kotlin you can set these properties in the constructor via `private val view: View` or `val view: View`

Reply:
Yeah! I'm actually wondering if I should use `RxView.clicks` here to get the click events as an `Observable` 🤔

Reply:
What would be the main benefit to switching to `RxView.clicks` rather than the current callback?

Reply:
Nothing in this case because I'm not doing anything special with the click event but it looks cool 😎 
It'd look like this 
```
RxView.clicks(this.view.reward_pledge_button)
      .compose(bindToLifecycle())
      .subscribe { this.viewModel.inputs.rewardClicked() }
```

Reply:
Sounds very cool, I say go for it!